lib/generators/suspenders/advisories_generator.rb

```diff
@@ -0,0 +1,26 @@
+module Suspenders
+  module Generators
+    class AdvisoriesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../templates/advisories", __FILE__)
+      desc(<<~TEXT)
+        Show security advisories during development.
+
+        Uses the `bundler-audit` gem and rake task to update the local security
+        database and show any relevant issues with the app's dependencies. This happens
+        on every test run and interaction with `bin/rake` and `bin/rails`.
+      TEXT
+
+      def add_bundler_audit
+        gem_group :development, :test do
+          gem "bundler-audit", ">= 0.7.0", require: false
+        end
+        Bundler.with_unbundled_env { run "bundle install" }
+      end
+
+      def configurea_rake_task
+        copy_file "bundler_audit.rake", "lib/tasks/bundler_audit.rake"
+        append_file "Rakefile", %(\ntask default: "bundle:audit"\n)
+      end
```

Comment:
This might require more thought. What if an application already has a default task? Additionally, what if we want to modify what we consider to be our default task in the future? I could imagine we'd want to run `standard` as part of our default task too?

Reply:
Now that these generators can be run independently, it can be challenging to piece them together. I wonder if instead we want a `rake` generator instead? Something that does the following:

- Runs `bundle:audit`
- Runs `standard`
  - [Lints HTML and ERB](https://github.com/thoughtbot/suspenders/pull/1107) is it's not an API application

This could also dovetail nicely with an CI generator, since I think there's value in keeping parity with the default Rake task and what's run in CI.

Reply:
We'll address this in #1144. For now, this generator will be responsible for installing the gem only.

bd01ede2bcebeff932ba19ed9907b4dbda36085e